src/Content.vala

```diff
@@ -62,10 +62,22 @@ public class ContentStack : Gtk.Stack {
             title = _("Yes."),
             result = "positive"
         },
+        Content () {
+            title = _("Just do it."),
```

Comment:
This doesn't really work; the user is not necessarily always asking if they should do something.

"Does Foobar love me?" "Just do it." is nonsensical. :D

Reply:
Ah ok I'll get rid of this